tests/foreman/sys/test_katello_certs_check.py

```diff
@@ -147,7 +147,9 @@ def validate_output(self, result, cert_data):
         assert set(options) == expected_result
 
     @pytest.mark.tier1
-    def test_positive_validate_katello_certs_check_output(self, cert_setup_teardown):
+    def test_positive_validate_katello_certs_check_output(
```

Comment:
I think this is a bad rebase, doubt you're trying to modify the fixture dependencies for this test.

Reply:
hmmm yeah. looks like this originally had more deps, but was modified by ??your?? change so I went the "safest" route during the rebase